app/src/ui/branches/ci-check-list-item.tsx

```diff
@@ -158,13 +161,21 @@ export class CICheckRunListItem extends React.PureComponent<
     )
   }
 
+  private hasActionsWorkflowLogs = (): boolean => {
```

Comment:
It doesn't appear as if we're calling this from an unbound context anywhere so we can probably just make it a regular class field

```suggestion
  private hasActionsWorkflowLogs() {
```